### PR TITLE
Updating CHANGELOG.md to add deprecation notes.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@markandrus

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,61 @@
...
+  });
+
+  participant.on('trackUnsubscribed', (track, publication) => {
+    console.log(`Unsubscribed from a RemoteTrack: ${track} ${publication}`);
+  });
+  ```
+
+### Migrating from RemoteTrack's deprecated properties and events
+
+- Instead of the RemoteTrack's `id` property, use its `sid` property.
```
Comment (markandrus) [inline CHANGELOG.md]: Recommendation: "use its `sid` or `name` property."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,61 @@
 1.12.0 (in progress)
 ====================
 
+Deprecations
+------------
+
+The following `1.x` APIs/events are now deprecated and scheduled for removal
+in `twilio-video.js@2.0.0`:
+
+- Participant's "trackAdded" and "trackRemoved" events
```
Comment (markandrus) [inline CHANGELOG.md]: Nice, btw, I think we need to update the usage example in README.md?

Comment (manjeshbhargav) [reply CHANGELOG.md]: @markandrus WDYT about updating README.md in the 2.x branch? I think we can do it once and for all there.

**Comment by markandrus** [comment]
+1

**Pull request merged**